tests/target/issue-5801/attribute_unexpectedly_wraps_before_max_width.rs

```diff
@@ -0,0 +1,18 @@
+// rustfmt-max_width: 120
+// rustfmt-version: One
+// rustfmt-attr_fn_like_width: 120
+
+pub enum Severity {
+    #[something(
+        AAAAAAAAAAAAA,
+        BBBBBBBBBBBBBB,
+        CCCCCCCCCCCCCCCC,
+        DDDDDDDDDDDDD,
+        EEEEEEEEEEEE,
+        FFFFFFFFFFF,
+        GGGGGGGGGGG
+    )]
+    AttrsWillWrap,
+    #[something_else(hhhhhhhhhhhhhhhh, iiiiiiiiiiiiiiii, jjjjjjjjjjjjjjj, kkkkkkkkkkkkk, llllllllllll, mmmmmmmmmmmmmm)]
+    AttsWontWrap,
```

Comment:
Let's fix the typo. ` AttsWontWrap` should be `AttrsWontWrap`

Reply:
Updated!